MethodsHomework/MethodsHomework.cs

```diff
@@ -166,8 +169,136 @@ public class YourAnswers
+        {
+            return 0 ;
+        }
+
+        public static int ReturnIntWithValue25()
+        {
+            return 25;
+        }
+
+        public static int AddTwoNumbers(int x, int y)
+        {
+            int addIt = x + y;
+            return addIt;
+        }
+
+        public static int IncrementMe(int x)
+        {
+            int y = x++;
+            return y;
+        }
+
+        public static int DecrementMe(int x)
+        {
+            int y = x--;
+            return y;
+        }
+
+        public static int CastDoubleToInt(double x)
+        {
+            double y = x;
+            int d = (int) y;
+            return d;
+        }
+
+        public static double ConvertIntToDouble(int x)
+        {
+            int y = x;
+            double d = (double) y;
+            return d;
+        }
+
+        public static string CastMeIntoAString(double x)
+        {
+            double y = x;
+            string str = x.ToString(); // intellisense is creeping me out
+            return str;
+        }
+
+        public static double ParseMeIntoADouble(string valueToBeParsed)
+        {
+            double result = double.Parse(valueToBeParsed);  // had no been parsed
+            return result; 
+        }
+
+        public static int ReturnMaxIntValue()
+        {
+                return int.MaxValue;
+        }
+
+        public static bool CheckIfIAmTrue(bool amITrue)
+        {
+            if (amITrue != true)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
+        public static bool ReturnTrueOnlyIfIamBothPositiveAndEven(int x)
+        {
+            if (x >= 0 && x  % 2 == 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
         
-    }
+
+        public static int SumTheContentsOfTheArray(int[] array)
+        {
+            int sum = 0;
+            for(int i = 0; i < array.Length; i++)
+                sum += i;
+            return sum;
+        }
+
+        public static int GetThirdValue(int[] array)
+        {
+            return array[2];
+        }
+
+        public static int GetLengthOfArray(int[] array)
+        {
+            return array.Length;
+        }
+
+        public static bool CheckAllValuesAre3(int[] array4)
+        {
+            
+            for (int i = 0; i < array4.Length; i++)
+            {
+                if (array4[i] != 3)
+                {
+                    return false;
+                }
+                else
+                {
+                    return true;
+                }
+                
+            }
+        }
+
+        public static int[] SwapFirstValueWithLastValue(int[] array4)
+        {
+            string swapped = array4[0];
+            array4[0] = array4[2];
+            array4[array4.Count - 1] = swapped;
```

Comment:
Arrays use Length. Lists use Count. It's confusing, but it's the way it is.
Even more confusing, arrays have a _method_ `Count()`. It would actually work if you add the parentheses, but it's part of LINQ, which we won't be covering for a while.